static/js/open-status-i18n.js

```diff
@@ -1,14 +1,12 @@
 import { DayNames } from './hours/constants.js';
 import { OpenStatusStrings } from './hours/open-status/constants.js';
-
+import { getLanguageFromLocale } from './utils'; 
 /**
  * Provides translation data for the openStatus formatter.
  * @param {string} locale
  */
 export default function provideOpenStatusTranslation (locale) {
-  const language = (locale === 'zh-CN' || locale === 'zh-TW') ? 
-    locale :
-    locale.substring(0,2);
+  const language = getLanguageFromLocale(locale);
```

Comment:
I have a PR open for the open status translations (https://github.com/yext/answers-hitchhiker-theme/pull/927) so this isn't needed here